src/common/utils.js

```diff
@@ -16,6 +17,16 @@ export function getAuthToken() {
   return result.json().accessToken;
 }
 
+export function getNoticesList(token){
```

Comment:
is this the same execution performed inside the notices/getNoticesList.js file? if yes, we could reuse it also there instead to copy/paste it

ps: maybe, instead to dirty this common file, you could move this function inside the notices/getNoticesList.js file itself (it's a something usefull just for the "notices" domain, I don't think we could use it outside that folder

Reply:
ok